### DISCUSSION TITLE
2023/11/19/%E6%97%A5%E6%9C%AC%E5%B9%B4%E8%BD%BB%E4%BA%BA%E4%B8%8D%E7%88%B1%E4%B8%8D%E5%A9%9A%E4%B8%8D%E7%94%9F%E5%AD%90%E6%98%AF%E4%B8%BA%E4%BA%86%E4%BD%93%E9%9D%A2/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#9] [General]
# 2023/11/19/%E6%97%A5%E6%9C%AC%E5%B9%B4%E8%BD%BB%E4%BA%BA%E4%B8%8D%E7%88%B1%E4%B8%8D%E5%A9%9A%E4%B8%8D%E7%94%9F%E5%AD%90%E6%98%AF%E4%B8%BA%E4%BA%86%E4%BD%93%E9%9D%A2/

有四分之一的年轻人对未来感到不安，是日本少子化的主要原因。要采取让年轻人不会对未来过不上富裕生活而感到不安的措施。 在东亚社会里，让子女接受比自己更好的教育，找到比自己更好的工作，是父母的生活意义所在。 如果现实恋爱的满足度下降了，那虚拟恋爱的价值就会上升。也许现在已经不能说现实中的满足就更优越了。   本文转载自南方周末公众号，发表日期为2023年11月17日

https://wz.anoms.top/2023/11/19/%E6%97%A5%E6%9C%AC%E5%B9%B4%E8%BD%BB%E4%BA%BA%E4%B8%8D%E7%88%B1%E4%B8%8D%E5%A9%9A%E4%B8%8D%E7%94%9F%E5%AD%90%E6%98%AF%E4%B8%BA%E4%BA%86%E4%BD%93%E9%9D%A2/

**Comment by Hazelandinstant** [comment]
“'与相爱的人在一起'就是人生的意义”,哇，这样的意义好浪漫呀，但是确实不适合东亚文化，因为东亚文化还包含了一个东西叫做“责任”，以及“集体主义”。

**Reply by cyclinbox** [reply]
确实，在东亚文化的社会谈恋爱，要考虑的东西太多了……
所以，“在不少人看来，谈恋爱其实就是寻找结婚对象的一种过程，越来越多的人认为，恋爱本身就是一种需要顾虑别人的不快乐的事”
